pkgs/top-level/ocaml-packages.nix

```diff
@@ -929,7 +929,14 @@ let
 
     ppxfind = callPackage ../development/ocaml-modules/ppxfind { };
 
-    ppxlib = callPackage ../development/ocaml-modules/ppxlib { };
+    ppxlib = callPackage ../development/ocaml-modules/ppxlib {
+      version =
+        if lib.versionAtLeast ocaml.version "4.12"
```

Comment:
```suggestion
        if lib.versionAtLeast ocaml.version "4.08"
```

Reply:
What's the reasoning for this? I tried to be conservative and only wanted to add the update when absolutely necessary (4.12) since when I last tried to update ppxlib to > 0.15.0 it broke a lot of stuff. Has the situation changed?